### ISSUE TITLE
Add contributing guidelines

### ISSUE THREAD

**Issue by marcelamelara** [in-toto/attestation#128]
Some guidelines are covered in the README (as of #126), but it would be great to have a more dedicated space to document these guidelines in more detail.

**Comment by adityasaky** [comment]
https://github.com/in-toto/community/blob/main/CONTRIBUTING.md

I wonder if we can just add a pointer to this.

**Comment by marcelamelara** [comment]
Good call, we probably want a pointer to the general contributing guidelines. I had something more like this in mind, though: https://github.com/marcelamelara/in-toto.attestation/blob/v1.0-spec-update/docs/new_predicate_guidelines.md

Thoughts?

**Comment by joshuagl** [comment]
We landed the new_predicate_guidelines in https://github.com/in-toto/attestation/pull/135 so we can perhaps close this. However, we _should_ also update the new predicate guidelines to ensure that they align with [ITE 9](https://github.com/in-toto/ITE/blob/master/ITE/9/README.adoc#specification).

As a separate issue, we should probably choose where the predicate definition format lives, in this repo or in ITE 9.
